tests/cpp_api/dataloader/dataloader.cpp

```diff
@@ -178,6 +178,21 @@ int main(int argc, const char **argv) {
             return -1;
         }
         rocalCopyToOutput(handle, mat_input.data, h * w * p);
+        if (!processing_device) {
+            float *f32_batch_output = (float *)aligned_alloc(8, 8 * ((inputBatchSize * h * w * p * sizeof(float)) / 8 + 1));
+            rocalToTensor(handle, f32_batch_output, RocalTensorLayout::ROCAL_NHWC, RocalTensorOutputType::ROCAL_FP32, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0, false, RocalOutputMemType::ROCAL_MEMCPY_HOST);
```

Comment:
I think this code is not valid as it is just calling all the same API with different results and overriding the memory. Though it might increase code coverage, it is not a good way to incorporate into a sample. You need to take args from user and call different variations depending on the use case.

Reply:
Added command line arguments to dataloader example to specify the to_tensor call configuration